manifests/params.pp

```diff
@@ -190,10 +197,7 @@
         }
       } elsif $::operatingsystem == 'Ubuntu' {
         $init_type = $::operatingsystemmajrelease ? {
```

Comment:
No we don't, i'll removed this. 

There are a few other variables in this params class that are unused. I'll try to clean them up the best i can.